bentoml/_internal/models/model.py

```diff
@@ -124,13 +149,20 @@ def flush_info(self):
         with self._fs.open(MODEL_YAML_FILENAME, "w") as model_yaml:
             self.info.dump(model_yaml)
 
+    def flush_custom_objects(self):
+        if self.custom_objects is None:
+            return
+        with self._fs.open(CUSTOM_OBJECTS_FILENAME, "wb") as cofile:
+            cloudpickle.dump(self.custom_objects, cofile)
```

Comment:
this part I want to make sure if user pass an empty dictionary, after loading the model from fs he/she will get back an empty dictionary